manifests/00-custom-resource-definition.yaml

```diff
@@ -1326,7 +1327,7 @@ spec:
                           type:
                             description: type is the underlying infrastructure provider
                               for the load balancer. Allowed values are "AWS", "Azure",
-                              "BareMetal", "GCP", "OpenStack", and "VSphere".
+                              "BareMetal", "GCP", "Nutanix", "OpenStack", and "VSphere".
                             enum:
```

Comment:
Is this crd yaml file generated? Why there is "Nutanix" added to the description, but not in the "enum" list?

Reply:
- This underlying source of the CRD is actually from the API repo.
- Hand edited the copied CRD here to test whether the change actually works

Reply:
https://github.com/openshift/api/blob/master/operator/v1/0000_50_ingress-operator_00-ingresscontroller.crd.yaml